src/btree/bt_vrfy_dsk.c

```diff
@@ -293,7 +293,7 @@ __verify_dsk_row(
 		case WT_CELL_ADDR_LEAF_NO:
 		case WT_CELL_KEY_OVFL:
 		case WT_CELL_VALUE_OVFL:
-			if (!bm->addr_valid(bm,
+			if (bm->addr_invalid(bm,
 			    session, unpack->data, unpack->size))
```

Comment:
I don't think so.

The only possible error returns (in non-diagnostic mode) are EINVAL and ENOMEM: `__wt_block_addr_invalid` calls `__wt_block_buffer_to_addr`, calls `__block_buffer_to_addr`, calls `__wt_vunpack_uint`, which returns either EINVAL or ENOMEM. Neither of those two errors means anything more than a corrupted value, as far as I can tell.

It would be great to try and defend against future code changes breaking this path, but I don't see any obvious way to do that. Suggestions?